src/components/Table/MainPot.tsx

```diff
@@ -1,59 +1,86 @@
 import { css } from "@emotion/core";
-import { useEffect, useState } from "react";
 import { Bet } from "../Chips";
 import { GameTurns } from "../../lib/constants";
 
 // This is the component that displays the main pot at the middle of table
 
 interface IProps {
-  mainPot: number;
+  pot: number[];
```

Comment:
Thanks, good point. Currently, since the game is heads-up only, we don't support side pots, at least at the front-end level.

@sg777 do we have side pot rules implemented at the backend? What are those?